docs/statistics.md

```diff
@@ -705,3 +718,86 @@ The value is used to apply TSBPD delay for reading the received data on the sock
 
 If `SRTO_TSBPDMODE` is off (default for **file mode**), 0 is returned.
 
+# SRT Group Statistics
+
+TODO:
+
+- Function to access group statistics + refer to Connection Bonding documentation
+- Check the types of statistics
+- Corresponding statistics in bytes
+- Interval-based + statistics in bytes
+
+## Summary Table <a name="group-summary-table"></a>
```

Comment:
The `<a>` tag isn't necessary to create jump links.

https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown

Reply:
I've added this tags for all the sections with repeated names. I already have "Summary Table" and other sections for individual socket statistics.

Reply:
I understand, but it looks ugly in this form in the raw text - and markdown is intended to be readable both as raw and as formatted text.